Fortran/gfortran/CMakeLists.txt

```
@@ -716,10 +716,18 @@ function(gfortran_add_execute_test expect_error main others fflags ldflags)
   gfortran_make_working_dir("${target}" working_dir)
   get_filename_component(working_dir_name "${working_dir}" NAME)
 
+  # When running in a emulator, use the version of 'not' that will
+  # spawn the subprocess under that emulator as well.
+  if(TEST_SUITE_USER_MODE_EMULATION AND TEST_SUITE_RUN_UNDER)
+    set(NOT_HELPER "not-spawning-emulator")
```

Perhaps name this variable `NOT_TOOL`. It's not really a helper.

```suggestion
    set(NOT_TOOL "not-spawning-emulator")
```